adhocracy4/phases/models.py

```diff
@@ -20,6 +20,29 @@ def active_phases(self):
     def finished_phases(self):
         return self.filter(end_date__lte=timezone.now())
 
+    def past_phases(self):
```

Comment:
I think this queryset API has now at least two problems: One they use different wording for features that sound very similar. One is <temporal_adjective>_phases  and the other <adverb>_phases (eg. past_phases vs finished_phases). And the second issue, those API endpoints partly duplicate code but also differ slightly in semantics (especially ordering). 

I think we should clear this up. Not saying we should do it right now, but it needs to be tracked somewhere.

While at it, we could also think whether we choose a smart default ordering for phases. But I guess we have to look back again why we choose it.